### PR TITLE
Little code clean up

### PR DESCRIPTION
**Pull request by arreche**
- Reviewed code style.
- Sorted methods. Android methods go first, then user actions and finally bus events.
- Added static ChatService assessor to Application.
- Started ChatService in background to pass the StrictMode.
- Ensured to handle UI events in the main thread.
- Refactored common stuff into a BaseActivity.

### CODE REVIEW

File: brewchat/src/main/java/com/example/brewchat/domain/User.java [hunk 1]
```diff
@@ -17,8 +17,7 @@ public User(int id, String name) {
     public int getId() { return id;}
 
     public String getName() { return name; }
-
-
+
```
Comment (JaySeeJC) [inline brewchat/src/main/java/com/example/brewchat/domain/User.java:20]: Git loves whitespace :D

**Comment by JaySeeJC** [comment]
Looks fantastic! +1

**Comment by jephillips** [comment]
Good work +1 

Perhaps we should create some style rules at some point to get everyone on the same page

**Comment by arreche** [comment]
Maybe we can use [Checkstyle](https://gradle.org/docs/current/userguide/checkstyle_plugin.html)

**Pull request merged**